tests/test-grad0.c

```diff
@@ -1154,7 +1208,7 @@ int main(int argc, const char ** argv) {
                             continue;
                         }
 
-                        struct ggml_tensor * f = ggml_sum(ctx0, ggml_rope(ctx0, x[0], n_past, n_rot, mode));
+                        struct ggml_tensor * f = ggml_sum(ctx0, ggml_rope(ctx0, x[0], n_past, n_rot, mode, 0));
```

Comment:
Is this right?

Reply:
Looks like it works, because:
- the test runs without error;
- and I saw `n_ctx` is set as `0` in other places.